### PR TITLE
fixes bug 1203669 - Drop time-stamp from Chapter display.

### PR DESCRIPTION
**Pull request by blossomica**
@peterbe - I attempted to fix this bug if there is a need for any changes feel free to let me know :)

### CODE REVIEW

File: airmozilla/main/templates/main/event.html [hunk 1]
```diff
@@ -177,7 +177,6 @@ <h1 class="entry-title">
               {% for chapter in chapters %}
               <li>
                 <a href="#@{{ show_duration_compact(chapter.timestamp) }}" data-ts="{{ chapter.timestamp }}">
```
Comment (peterbe) [inline airmozilla/main/templates/main/event.html]: Add a title attribute to this tag. E.g `title="{{ show_duration(chapter.timestamp, include_seconds=True) }}"`

**Comment by blossomica** [comment]
@peterbe - Just added it.

**Comment by peterbe** [comment]
r+ 

Well done!

**Comment by blossomica** [comment]
@peterbe - Thank you! All is set now.

**Pull request merged**